### PR TITLE
[995] Add specific circumstances page

### PR DESCRIPTION
**Pull request by jamesrose**
### Context

This is the first PR of https://trello.com/c/pualSpPd/955-update-content-around-mno-offer-on-school-journey

### Changes proposed in this pull request

- Add a specific circumstances page
- Link to given page on the homepage if the school is within the MNO feature.
- Page links to existing 'devices' and 'extra data for mobile devices' request pages for circumstances, the latter of which will be updated in a subsequent PR. 

### Guidance to review

1. Login as user belonging to a school within the MMO feature
2. New page should be linked to from homepage

1. Login as user belonging to a school not within the MMO feature
2. New page should not be linked to from homepage

### Screenshots

screencapture-localhost-3000-schools-100000-specific-circumstances-2020-11-05-16_04_45

screencapture-localhost-3000-schools-100000-2020-11-05-16_05_25

### CODE REVIEW

**Review by benilovj — APPROVED** [review]
LGTM, @fofr can review this from a design perspective

File: spec/features/school/specific_circumstances_spec.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    end
+  end
+
+  context 'when the MNO offer is not activated' do
+    before do
+      school.update!(mno_feature_flag: false)
+      sign_in_as user
+    end
+
+    scenario 'the user can navigate to the specific circumstances page from the home page' do
```
Comment (fofr) [inline spec/features/school/specific_circumstances_spec.rb:41]:
```suggestion
    scenario 'the user cannot navigate to the specific circumstances page from the home page' do
```

**Review by fofr — APPROVED** [review]
Small tweak to fix but 👍

File: spec/features/school/specific_circumstances_spec.rb (hunk 1 above)
Comment (jamesrose) [reply spec/features/school/specific_circumstances_spec.rb:41]: Good spot! Thanks.

**Pull request merged**